### PR TITLE
Add flag to ignore POT-Creation-Date for updates

### PR DESCRIPTION
**Pull request by joeportela**
Adding a flag to ignore changes to POT-Creation-Date when performing updates or checks. 
I use a CI check that ensures my .po files are updated before merging, but there is always a diff on POT-Creation-Date.

Related to https://github.com/python-babel/babel/issues/725 - but solving (my) problem via update instead of extract.

### CODE REVIEW

**Comment by joeportela** [comment]
Ping @akx 
Any interest in this PR?

File: babel/messages/catalog.py [hunk 1]
```diff
@@ -769,6 +769,7 @@ def update(
         no_fuzzy_matching: bool = False,
         update_header_comment: bool = False,
         keep_user_comments: bool = True,
+        ignore_pot_creation_date: bool = False,
```
Comment (akx) [inline babel/messages/catalog.py:772]: I think

```suggestion
        update_creation_date: bool = True,
```

would be more intuitive? 

The command line parameter could be left `--ignore-pot-creation-date` though.

**Review by akx — CHANGES_REQUESTED** [review]
Thank you for the contribution! As mentioned inside, I think it could be more intuitive for the kwarg to be positive (and default to True).

**Comment by joeportela** [comment]
> Thank you for the contribution! As mentioned inside, I think it could be more intuitive for the kwarg to be positive (and default to True).

My pleasure, thanks for taking a look! Just updated per your suggestion.

**Review by akx — APPROVED** [review]
LGTM, sorry for the delay!

**Pull request merged**